Assets/Scripts/Battle/EnemyEncounter.cs

```diff
@@ -104,20 +120,18 @@ public class EnemyEncounter : MonoBehaviour {
         DynValue enemyPositionsLua = script.GetVar("enemypositions");
         string musicFile = script.GetVar("music").String;
 
-        try { enemies = new EnemyController[enemyScriptsLua.Table.Length]; /*dangerously assumes enemies is defined*/ }
-        catch (Exception) {
-            UnitaleUtil.DisplayLuaError(StaticInits.ENCOUNTER, "There's no enemies table in your encounter. Is this a pre-0.1.2 encounter? It's easy to fix!\n\n"
-                + "1. Create a Monsters folder in the mod's Lua folder\n"
-                + "2. Add the monster script (custom.lua) to this new folder\n"
-                + "3. Add the following line to the beginning of this encounter script, located in the mod folder/Lua/Encounters:\nenemies = {\"custom\"}\n"
-                + "4. You're done! Starting from 0.1.2, you can name your monster and encounter scripts anything.");
+        if (enemyScriptsLua.Table == null) {
+            UnitaleUtil.DisplayLuaError(StaticInits.ENCOUNTER, "There has to be an enemies table in your encounter's file.");
```

Comment:
Good idea to remove that 'quickfix' text, I'm sure it's more than obsolete by now